docs/cis_self_assessment.md

```diff
@@ -753,7 +630,7 @@ on the master node and set the `--authorization-mode` parameter to values other
 One such example could be as below.
 
 ``` bash
---authorization-mode=RBAC
+--authorization-mode=Node,RBAC
```

Comment:
Assuming more changes for the rest of this control and it will be revisited when you get to this particular control.